src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GeneralFeedback/hoots.test.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
```

Comment:
Will you update the file name to `hooks.test.js`

Reply:
No this file is for owls! just kidding! will fix